src/corehost/common/pal.windows.cpp

```diff
@@ -215,6 +215,77 @@ bool pal::get_local_dotnet_dir(pal::string_t* dir)
     return true;
 }
 
+// To determine the OS version, we are going to use RtlGetVersion API
+// since GetVersion call can be shimmed on Win8.1+.
+typedef NTSTATUS (WINAPI *pFuncRtlGetVersion)(RTL_OSVERSIONINFOW *);
+
+pal::string_t pal::get_current_os_rid_platform()
+{
+    pal::string_t ridOS;
+    
+    RTL_OSVERSIONINFOW osinfo;
+
+    // Init the buffer
+    ZeroMemory(&osinfo, sizeof(osinfo));
+    osinfo.dwOSVersionInfoSize = sizeof(osinfo);
+    HMODULE hmodNtdll = LoadLibrary("ntdll.dll");
```

Comment:
We cannot since the import library in question is not part of the SDK (or guaranteed to be part of it).